nsls2forge_utils/dashboard.py

```diff
@@ -8,31 +8,25 @@
 from .meta_utils import get_attribute
 
 
-def _extract_org(pkg):
-    try_dev_url = False
-    # get org from home url
-    home_str = get_attribute('about home', pkg, 'nsls-ii-forge')
-    if home_str is None:
-        try_dev_url = True
-    elif 'github.com' in home_str:
-        split_str = home_str.split('/')
+def _extract_github_org_from_url(url):
+    if url is not None and 'github.com' in url:
+        split_str = url.rstrip('/').split('/')
```

Comment:
I am still thinking it can be done better with the specialized libraries, such as `urllib.parse.urlparse` (which is a part of standard Python library):
```py
2020-08-04T12:29:58.945336 [14] ▶ from urllib.parse import urlparse

2020-08-04T12:30:02.854692 [15] ▶ urlparse?
Signature: urlparse(url, scheme='', allow_fragments=True)
Docstring:
Parse a URL into 6 components:
<scheme>://<netloc>/<path>;<params>?<query>#<fragment>
Return a 6-tuple: (scheme, netloc, path, params, query, fragment).
Note that we don't break the components up in smaller bits
(e.g. netloc is a single string) and we don't expand % escapes.
File:      ~/anaconda3/envs/conda-smithy/lib/python3.7/urllib/parse.py
Type:      function

2020-08-04T12:30:16.322948 [16] ▶ urlparse('https://github.com/org/repo/')
Out[16]: ParseResult(scheme='https', netloc='github.com', path='/org/repo/', params='', query='', fragment='')

2020-08-04T12:30:24.849750 [17] ▶ urlparse('https://github.com/org/repo/').path
Out[17]: '/org/repo/'

2020-08-04T12:31:48.976390 [23] ▶ urlparse('https://github.com/org/repo/').path.strip('/').split('/')
Out[23]: ['org', 'repo']

2020-08-04T12:31:56.682498 [24] ▶ urlparse('https://github.com/org/repo').path.strip('/').split('/')
Out[24]: ['org', 'repo']

2020-08-04T12:32:01.654053 [25] ▶ urlparse('https://github.com/org/').path.strip('/').split('/')
Out[25]: ['org']
```
That way you will always get a `path`, which can be empty or have one element.